kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -208,6 +209,8 @@ const List: React.FC<TopicsListProps> = ({
 
       const clearTopicMessagesHandler = React.useCallback(() => {
         clearTopicMessages(clusterName, name);
+        fetchTopicsList(fetchTopicsListParams);
+        setClearMessagesConfirmationVisible(false);
       }, [name]);
```

Comment:
shouldn't fetchTopicsListParams be in the dependency ?

Reply:
Right, missed that.